CMakeLists_files.cmake

```diff
@@ -64,6 +64,7 @@ list (APPEND TEST_SOURCE_FILES
 	tests/cpgrid/orientedentitytable_test.cpp
```

Comment:
> There are lots more files added, so this list must be updated.

I was confused by that too, but if you look closely there is a separate `CMakeLists.txt` in the `dune/grid/polyhedralgrid` directory that **does** list all additional files.  I have not looked into how the two systems interact and I'd appreciate some insight from a build-system expert.